cmake_files/dependencies.cmake

```diff
@@ -19,7 +19,9 @@ endif()
 
 find_package(CFitsIO REQUIRED)
 
-cmake_policy(SET CMP0167 OLD)
+if(CMAKE_VERSION VERSION_GREATER_EQUAL "3.29.0")
+  cmake_policy(SET CMP0167 NEW)
+endif()
```

Comment:
Should be `VERSION_GREATER_EQUAL "3.30.0"`?

https://cmake.org/cmake/help/latest/policy/CMP0167.html

Reply:
Yes! Misread, sorry. Fixed now.